### PR TITLE
rpc: Mention getwalletinfo where a rescan is triggered

### PR DESCRIPTION
**Pull request by promag**

### CODE REVIEW

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -1359,7 +1364,8 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
                 "If an address/script is imported without all of the private keys required to spend from that address, it will be watchonly. The 'watchonly' option must be set to true in this case or a warning will be returned.\n"
                 "Conversely, if all the private keys are provided and the address/script is spendable, the watchonly option must be set to false, or a warning will be returned.\n"
             "\nNote: This call can take over an hour to complete if rescan is true, during that time, other rpc calls\n"
-            "may report that the imported keys, addresses or scripts exists but related transactions are still missing.\n",
+            "may report that the imported keys, addresses or scripts exists but related transactions are still missing.\n"
```
Comment (thijstriemstra) [inline src/wallet/rpcdump.cpp]: use "exist" insead of "exists" here.

Comment (promag) [reply src/wallet/rpcdump.cpp]: Done.

**Comment by jonasschnelli** [comment]
Some redudant strings, but I guess that's okay.
utACK be4efb165ab7ff5824de9a46aa06eca654b7513f

**Comment by sipa** [comment]
utACK be4efb165ab7ff5824de9a46aa06eca654b7513f

**Pull request merged**